lib/Quantization/Quantization.cpp

```diff
@@ -226,10 +226,11 @@ static Node *quantizeNode(Function *F, Node *node,
     break;                                                                     \
   }
     CASE_QUANTIZE_NODE(Add);
-    CASE_QUANTIZE_NODE(Mul);
-    CASE_QUANTIZE_NODE(Sub);
+    CASE_QUANTIZE_NODE(Div);
```

Comment:
LGTM. Can you run some networks containing "div" through the quantization procedure?

Reply:
Didn't break with densenet with cpu and opencl